prow/manifests/overlays/metal3/config.yaml

```diff
@@ -542,7 +533,15 @@ presubmits:
       agent: jenkins
       # Don't run unless triggered to avoid wasting resources
       always_run: false
-      # Until we have checked that it works, keep it optional
+      optional: false
```

Comment:
What do you think? Should we trigger it automatically instead but have 
```
skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
```

Reply:
I think this is the way to go. We've spent quite a lot of time developing the e2e test to make BMO testing more standalone, and its rather fast and stable test, so why wouldn't we want to run it?